### PR TITLE
Dev add inscription relationship

### PR DESCRIPTION
**Pull request by sflorezs1**
Added some missing relationships between classes, and implemented the inscription system.

### CODE REVIEW

**Review by GrayDiamond493 — APPROVED** [review]

File: resources/views/courses/show.blade.php [hunk 1]
```diff
@@ -10,8 +10,18 @@
...
+                    
+                    <form action="{{ route('inscription.enroll') }}" method="post">
+                        @csrf
+                        <input type="hidden" id="courseId" name="courseId" value="{{ $data['course']->getId() }}">
+                        <button type="submit" class="btn btn-sm btn-primary">
+                            <i class="fa fa-check"></i> @lang('messages.course.list.enroll.submit')
+                    </form>
+                </div>
                 <div class="card-body">
                     <h5><span class="font-weight-bold">@lang("messages.course.id"):</span>
```
Comment (jandrovins) [inline resources/views/courses/show.blade.php:26]: Around these lines should be listed the user author of the course too.

File: app/Models/Inscription.php [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 class Inscription extends Model
 {
-    // 
+    //
```
Comment (jandrovins) [inline app/Models/Inscription.php:9]: Remember to add the comment listing all attributes of this class.

File: app/Models/User.php [hunk 3]
```diff
@@ -12,7 +12,7 @@ class User extends Authenticatable
 
     /**
      * The attributes that are mass assignable.
-     * 
+     *
      * @var array
      */
```
Comment (jandrovins) [inline app/Models/User.php:17]: I guess the bold developers of Laravel do not add the comment listing all attributes of this class. Could you add it?

**Review by jandrovins — CHANGES_REQUESTED** [review]
I leaved some comments on specific suggestions. Let me know your thoughts.

Every change not mentioned is OK with me.

**Review by jandrovins — APPROVED** [review]

**Pull request merged**